1.10.7/alpine3.10/.trivyignore

```diff
@@ -0,0 +1,8 @@
+# ignore in old airflow versions; it's fixed ONLY in new airflow version
+# lodash: Prototype pollution in utilities function fixed in >=4.17.11
+CVE-2018-16487
```

Comment:
Do we still need this one for 1.10.7? I forget whether we pulled the UI fixes back to.

Reply:
```
#!/bin/sh -eo pipefail
trivy --ignorefile ./1.10.7/alpine3.10/trivyignore --cache-dir /tmp/workspace/trivy-cache --ignore-unfixed -s HIGH,CRITICAL --exit-code 1 --no-progress "astronomerinc/ap-airflow:1.10.7-alpine3.10-onbuild"
2020-03-16T20:08:58.762Z	INFO	Need to update DB
2020-03-16T20:08:58.762Z	INFO	Downloading DB...
2020-03-16T20:08:59.663Z	INFO	Reopening DB...
2020-03-16T20:09:22.366Z	INFO	Detecting Alpine vulnerabilities...
2020-03-16T20:09:22.367Z	INFO	Detecting yarn vulnerabilities...

astronomerinc/ap-airflow:1.10.7-alpine3.10-onbuild (alpine 3.10.4)
==================================================================
Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)


usr/lib/python3.7/site-packages/airflow/www_rbac/yarn.lock
==========================================================
Total: 1 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 1, CRITICAL: 0)

+---------+------------------+----------+-------------------+---------------+--------------------------------+
| LIBRARY | VULNERABILITY ID | SEVERITY | INSTALLED VERSION | FIXED VERSION |             TITLE              |
+---------+------------------+----------+-------------------+---------------+--------------------------------+
| lodash  | CVE-2018-16487   | HIGH     | 3.10.1            | >=4.17.11     | lodash: Prototype pollution in |
|         |                  |          |                   |               | utilities function             |
+---------+------------------+----------+-------------------+---------------+--------------------------------+

Exited with code exit status 1
```